src/main/kotlin/network/warzone/mars/player/achievements/AchievementDebugger.kt

```diff
@@ -0,0 +1,22 @@
+package network.warzone.mars.player.achievements
+
+import org.bukkit.Bukkit
+import org.bukkit.ChatColor
+import org.bukkit.entity.Player
+
+@Deprecated("Remove this")
+interface AchievementDebugger {
```

Comment:
Could you please explain the need for this interface?

Reply:
Can be removed

Reply:
This interface is paramount to the successful functionality of the achievements system, as it contains my username and thus etches my mark into the bowels of Mars for all eternity. 

Removing this interface may have serious consequences, such as the complete and utter implosion of the Mars codebase. Think of it as one of those warranty seals where once you break it off, your warranty is no longer valid.

So to answer your question: there is no need for this interface.